src/js/jsx/sections/libraries/LibrariesPanel.jsx

```diff
@@ -110,89 +108,96 @@ define(function (require, exports, module) {
         _handleLibraryRemove: function () {
             this.getFlux().actions.libraries.removeCurrentLibrary();
         },
-        
+
         /**
          * Return library panel content based on the connection status of CC Library.
          * @private
-         * 
+         *
          * @return {ReactComponent}
          */
         _renderLibrariesContent: function () {
+            if (!this.props.visible || this.props.disabled) {
```

Comment:
Not sure how we do this everywhere else... but I think it makes the code much more easy to understand when render functions don't return null. Can you move this check to where we call `renderLibrariesContent`? 

This way, another engineer reading the `render` function will not be surprised by something returning null.

If you disagree with me, that is fine too. But then update the JSDoc to signify that it may return null.